src/api/java/com/minecolonies/api/configuration/CommonConfiguration.java

```diff
@@ -485,6 +486,11 @@ protected CommonConfiguration(final ForgeConfigSpec.Builder builder)
                    ("minestuck:ore_cruxite",
                      "minestuck:ore_uranium"),
           s -> s instanceof String);
+
+        disabledModTags = defineList(builder, "disabledmodtags",
+          Collections.singletonList("minecolonies"),
```

Comment:
Not disable tags from all mods.
But allow tags to be specifically excluded.

See Aequivaleos vanilla plugin on how todo that, since that already filters on tags.

Reply:
Meh, that would become a large list we always have to extend every time we add a new crafter. I rather not do that.

Reply:
I rather you would. This already makes no sense since minecraft itself has both collective and functional tags.

So make the distinction on tag level please.
Possibly with a mod wide override.

Reply:
I'll make it opt-in, I think that's the best thing anyway.